src/awesome_map/EventSynthesizer.js

```diff
@@ -317,7 +317,7 @@ define(function(require) {
                         pageX: event.pageX,
                         pageY: event.pageY
                     },
-                    srcEvent: event.source
+                    srcEvent: event.source || event
```

Comment:
You're using `event.source || event` because we may not know what type of event this is (Hammer or Browser). But it seems like in most of these handlers we probably *do* know what the event type is because we've registered for the events in `EventSynthesizer#_initialize`. If we know what the event type is that a handler is consuming, we should just set the srcEvent to it rather than using the 'or' statement.